### PR TITLE
Export atom button sizes and designs the right way

### PR DESCRIPTION
**Pull request by jordevo**

### CODE REVIEW

**Review by davidmartin84 — APPROVED** [review]

File: components/atom/button/src/index.js [hunk 1]
```diff
@@ -228,6 +225,7 @@ AtomButton.propTypes = {
 
 export default AtomButton
 export {GROUP_POSITIONS as atomButtonGroupPositions}
-export {TYPES as atomButtonTypes}
 export {COLORS as atomButtonColors}
 export {DESIGNS as atomButtonDesigns}
+export {SIZES as atomButtonSizes}
+export {TYPES as atomButtonTypes}
```
Comment (rmoralp) [inline components/atom/button/src/index.js:231]: Si alguien utiliza el formato Array le romperás la compatibilidad. DESIGNS cambia el tipo 🤔

Comment (rmoralp) [reply components/atom/button/src/index.js:231]: Es posible que nadie lo use porque con un array es un poco "meh", pero podrías comprobarlo antes?

**Review by andresin87 — APPROVED** [review]

**Review by andresin87 — COMMENTED** [review]
Take a look on the retrocompatibility for the exported Arrays values changes

File: components/atom/button/src/index.js (hunk 1 above)
Comment (jordevo) [reply components/atom/button/src/index.js:231]: `DESIGNS` no se exportaba hasta hoy ;)

https://github.com/SUI-Components/sui-components/pull/1132/files

**Comment by jordevo** [comment]
@andresin87 the `DESIGNS` constant wasn't being exported until today with a previous PR
https://github.com/SUI-Components/sui-components/pull/1132/files

on this PR I'm only fixing what we were exporting to make it easier to use

**Review by devictoribero — APPROVED** [review]
`Designs` was not exported until now, there's no external possibility to break something.

If internally it works the same, PR is ready to be merged

**Review by andresin87 — APPROVED** [review]

**Pull request merged**